app/views/deals/_deal.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="is-flex has-background-color-white mx-2 mt-1 p-2 card-shadow">
+  <div class="is-flex flex-grow-1 flex-direction-column pl-2">
+    <div class="is-flex justify-space-between my-2">
+      <h5 class="fw-regular is-flex flex-grow-1"><%= deal.name %></h5>
+      <small class="fw-bold">$<%= deal.amount.round(2) %></small>
+    </div>
+    <small><%= deal.created_at.strftime("%d %b %Y") %></small>
+  </div>
+</div>
```

Comment:
- [ ] **[OPTIONAL]** When there are no transactions created, I suggest you add a friendly message that tells the user that there are no transactions added yet. This will make your app more user-friendly.